scripts/chroot.sh

```diff
@@ -30,33 +30,36 @@ if [ -n "${KEY}" ]
 then
     echo "Adding APT key: ${KEY}"
     apt-key add "${KEY}"
+    echo $KEY
 fi
 
 # Add all distro PPAs
-if [ $# -gt 0 ]
+if [ -n "${LIVEBRANCHES}" ] && [-n "${CODENAME}"]
```

Comment:
I believe this isn't  correct bash syntax? There should be spaces, `[ -n "${CODENAME}" ]`, or it will complain `[-n: command not found`.